Yona/fabric.apikey

```diff
@@ -0,0 +1 @@
+
```

Comment:
As said above, remove this file from Git. Just create it locally as an untracked file. You can even put the "official" API key in it if you want, but don't add it to Git.

Reply:
If we create file(fabric.apikey) through shell command in workspace, Xcode won't add file into target,  hence this will not be copied into the ipa file during archiving. That's the reason we created empty file(which is part of target) into repo and writing into it before build. Also added that into .gitignore to avoid comitting back the keys to GIT repo.
There are some third-pary libraries which can add files to xcode target from cli, which we may need to test before using here.
https://github.com/zackslash/Xcode-Proj-Adder/blob/master/README.md